tests/test_lib_utils.py

```diff
@@ -973,3 +973,31 @@ def test_34_to_list(self):
         r = to_list({"Hallo", "Du", "da"})
         self.assertIsInstance(r, list)
         self.assertEqual(3, len(r))
+
+    def test_35_parse_string_to_dict(self):
+        # We can have as many whitespaces
+        d = parse_string_to_dict(" :key1: v1 v2  v3:key2: v4 v5 ")
+        self.assertIn("key1", d)
+        self.assertIn("key2", d)
+        self.assertEqual(d.get("key1"), ["v1", "v2", "v3"])
+        self.assertEqual(d.get("key2"), ["v4", "v5"])
+
+        d = parse_string_to_dict(" :key1: v1 v2  *:*: v4 v5 ")
+        self.assertIn("key1", d)
+        self.assertIn("*", d)
+        self.assertEqual(d.get("key1"), ["v1", "v2", "*"])
+        self.assertEqual(d.get("*"), ["v4", "v5"])
+
+        # we can hove no whitespaces
+        d = parse_string_to_dict(":key1:v1 v2 v3 :key2:v4 v5")
+        self.assertIn("key1", d)
+        self.assertIn("key2", d)
+        self.assertEqual(d.get("key1"), ["v1", "v2", "v3"])
+        self.assertEqual(d.get("key2"), ["v4", "v5"])
+
+        # side effect: we can skip the leading colon
+        d = parse_string_to_dict("key1:v1 v2 v3 :key2:v4 v5")
+        self.assertIn("key1", d)
+        self.assertIn("key2", d)
+        self.assertEqual(d.get("key1"), ["v1", "v2", "v3"])
+        self.assertEqual(d.get("key2"), ["v4", "v5"])
```

Comment:
add newline, please

Reply:
This seems outdated. Newline and new tests exist. No noeed for action.